packages/components/src/templates/next/components/internal/Wizgov/WizgovWidget.tsx

```diff
@@ -0,0 +1,14 @@
+import type { WizgovWidgetProps } from "~/interfaces"
+import { WizgovWidgetClient } from "./WizgovWidgetClient"
+
+// Reference: https://github.com/opengovsg/wizgov
+export const WizgovWidget = (props: WizgovWidgetProps) => {
+  const { site, ...rest } = props
+
+  const scriptUrl =
+    site.environment === "production"
+      ? "https://script.wiz.gov.sg/widget.js"
+      : "https://script-staging.wiz.gov.sg/widget.js"
```

Comment:
FYI, we need to add this for our CSP!

Reply:
yup! there's a corresponding PR for that https://github.com/opengovsg/isomer-next-infra/pull/102